src/cli/index.js

```diff
@@ -89,9 +90,23 @@ try {
 const questions = [
   {
     type: 'list',
+    pageSize: 10,
     name: 'template',
     message: 'InstantSearch template',
-    choices: getAllTemplates(),
+    choices: () => {
+      const templatesByCategory = getTemplatesByCategory();
+      const choices = [];
+
+      for (const [category, values] of Object.entries(templatesByCategory)) {
```

Comment:
You can also use a `reduce` rather than the for loop.

Reply:
I first wanted to use `reduce` but it seemed like it was too much compared to this loop. I will compare them again.

Reply:
This would be the equivalent:

```javascript
Object.entries(templatesByCategory).reduce(
  (templates, [category, values]) => [
    ...templates,
    new inquirer.Separator(category),
    ...values,
  ],
  []
);
```

Not sure it more readable though.

Reply:
Never mind it's not a big deal.